Database/set up

ssbyrne89 opened the pull request:
# Pull Request Template

## Description

_Please include a summary of the change and/or which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change._

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist: 

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

ssbyrne89 committed:
Update db.py

added a class model to start the first table of the database

ssbyrne89 committed:
Update main.py

had to change the imports slightly to make it work locally

ssbyrne89 committed:
Update ocr.py

had to change the import to work locally

ssbyrne89 committed:
Update viz.py

added a dummy graph to work on diplaying a visual

ssbyrne89 committed:
new import

had to import pylint to stop errors from appearing

ssbyrne89 committed:
Update db.py

finished writing the database table models

jproctor-rebecca reviewed (approved):
do we need to hard code pdf_file CaseObject or would it be better to use a 'file_path_name'?  (line32)

jproctor-rebecca reviewed (approved):
was there a specific reason for the recording of title, description, and docs_url in app = FastAPI()?

ssbyrne89 committed:
db files added

model.py and schemas.py were added to the app folder to create the

jproctor-rebecca commented on app/db.py:
```diff
@@ -30,8 +30,28 @@ class CaseObject(BaseModel):
     outcome: Optional[str] = "Granted"
     country_of_origin: Optional[str] = "Kazakhstan"
     pdf_file: Optional[str] = 'C://file/some/where'
```
is this hard coding example of the path temporary? Since we seem to use this same file path in a lot of different files, what do you think about using a string like 'file_path' instead so the next team only has to change it in one place - it can be imported to the files it's used?

ssbyrne89 replied:
can do. and I made that change.

ssbyrne89 committed:
database connection and endpoint set-up

connected to the asylum database and created the endpoints to interact with the database

ssbyrne89 committed:
consolidated into db.py

ssbyrne89 committed:
consolidated into db.py

tomashphill reviewed (approved):
It looks good to me.
